openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
+require "#{File.dirname(__FILE__)}/resources/os_lib_reporting"
+require "#{File.dirname(__FILE__)}/resources/os_lib_schedules"
+require "#{File.dirname(__FILE__)}/resources/os_lib_helper_methods"
 
 #start the measure
-class StandardReports < OpenStudio::Ruleset::ReportingUserScript
+class OpenStudioResults < OpenStudio::Ruleset::ReportingUserScript
```

Comment:
Hmm, now that I look at this you seem to have overwritten the StandardReports measure with this new one.  I thought you were adding a new measure and not overwriting this one?   Now the class name does not match the file name.  You did seem to give this a new UUID.